server/route.go

```diff
@@ -1949,12 +1950,12 @@ func (c *client) processRouteConnect(srv *Server, arg []byte, lang string) error
 		c.closeConnection(WrongGateway)
 		return ErrWrongGateway
 	}
-	var perms *RoutePermissions
-	//TODO this check indicates srv may be nil. see srv usage below
-	if srv != nil {
-		perms = srv.getOpts().Cluster.Permissions
+
+	if srv == nil {
+		return errors.New("server is not set")
```

Comment:
Maybe use `ErrServerNotRunning` or add a new item in errors.go.

Reply:
Thanks. Changed to `ErrServerNotRunning`